paddle/fluid/operators/ngraph/ops/batch_norm_op.h

```diff
@@ -44,7 +44,8 @@ void BuildBatchNormNode(
   auto x = paddle::platform::GetInputNode(op, "X", ngb_node_map);
```

Comment:
Why do you update `batch_norm_op.h` and `conv2d_op.h` here? They are not related to this PR.

Reply:
On the whole, these changes are necessary for the CAPI tests - to pass. I agree with you that it is not associated with this PR, but these changes are really small. If you want I can move it to separate PR - but it will be generated any additional time.

Reply:
depthwise_conv can be done in a separate PR (https://github.com/PaddlePaddle/Paddle/pull/17454). The batchnorm update is to pass the test due to the BN fast math issue. That is the easiest way to make it align with paddle precision.